client/data/settings/selectors.js

```diff
@@ -38,7 +38,7 @@ export const getAvailablePaymentMethodIds = ( state ) => {
 };
 
 export const getPaymentMethodStatuses = ( state ) => {
-	return getSettings( state ).payment_method_statuses || EMPTY_ARR;
```

Comment:
I noticed that the return type of `getPaymentMethodStatuses` is not an array, but a key-value-pair, by default.